src/primitives/block.h

```diff
@@ -25,7 +39,7 @@ class CBlockHeader
 public:
     static const int32_t CURRENT_VERSION=SIZE_FORK_VERSION;
     // This code knows the rules for these block versions:
-    static const int32_t UNDERSTOOD_VERSIONS = (SIZE_FORK_VERSION | 3 | 2 | 1);
+    static const int32_t UNDERSTOOD_VERSIONS = (SIZE_FORK_VERSION | 4 | 3 | 2 | 1);
```

Comment:
But 3 is not a bit. It's two bits. What was 3 previously used for and wouldn't it have been confused with 1 and 2?